solver/llbsolver/vertex.go

```diff
@@ -79,17 +79,21 @@ func NormalizeRuntimePlatforms() LoadOpt {
 					OS:           p.OS,
 					Architecture: p.Architecture,
 					Variant:      p.Variant,
+					OSVersion:    p.OSVersion,
+					OSFeatures:   p.OSFeatures,
 				}
 			}
 			op.Platform = defaultPlatform
 		}
-		platform := ocispecs.Platform{OS: op.Platform.OS, Architecture: op.Platform.Architecture, Variant: op.Platform.Variant}
+		platform := ocispecs.Platform{OS: op.Platform.OS, Architecture: op.Platform.Architecture, Variant: op.Platform.Variant, OSVersion: op.Platform.OSVersion, OSFeatures: op.Platform.OSFeatures}
```

Comment:
nit: since the line is getting till col 198, would be nice to make this multiline for readability?

Reply:
Makes sense. I kind-of assumed gofmt would take care of that.